doc/en/example/reportingdemo.rst

```diff
@@ -9,9 +9,9 @@ and how ``pytest`` presents things (unfortunately
 not showing the nice colors here in the HTML that you
 get on the terminal - we are working on that)::
 
-    $ pytest failure_demo.py
+    assertion $ pytest failure_demo.py
```

Comment:
That fixed a regression in Regen introduced by me in another commit: it means regen should change to the `assertion` subdirectory before executing the command.